routes/interactions.js

```diff
@@ -13,6 +14,17 @@ router.get("/user/:id", async (req, res) => {
   });
 });
 
+router.get("/channel/:id", async (req, res) => {
```

Comment:
do you think about should use the interaction controller?

Reply:
hmm sounds good. doing it right now